actdocs/templates/index/whatis

```diff
@@ -5,22 +5,23 @@
       <h3>Barcelona</h3>
       <p class="lead">
 		The place to be.
+    Saturday Nov 5th. Plaça Catalunya 1 3D. Get there by <a href="https://www.google.es/maps/dir/Terminal+T1,+El+Prat+de+Llobregat/CAPSiDE,+Barcelona/@41.3365848,2.0440368,12z/data=!3m1!4b1!4m13!4m12!1m5!1m1!1s0x12a49e71d94a68c3:0x4135e11f17bbdf78!2m2!1d2.0746423!2d41.289182!1m5!1m1!1s0x12a4a27c6193b1a7:0x60a6240b19be359b!2m2!1d2.16937!2d41.386066">Plane</a>, <a href="https://www.google.es/maps/dir/Estaci%C3%B3n+de+Barcelona+Sants,+Carrer+del+Rector+Triad%C3%B3,+Barcelona/CAPSiDE,+Barcelona/@41.3823556,2.1463693,15z/data=!3m1!4b1!4m13!4m12!1m5!1m1!1s0x12a49880d4cf5639:0x3a7c2d44a50f178f!2m2!1d2.1401328!2d41.3790928!1m5!1m1!1s0x12a4a27c6193b1a7:0x60a6240b19be359b!2m2!1d2.16937!2d41.386066">Train</a>, <a href="https://www.tmb.cat/">Bus</a>, <a href="http://www.fgc.cat/cat/index.asp">FCC</a> or <a href="https://www.tmb.cat/">Metro</a>. The event is <b>FREE</b>
 	  </p>
     </div>
         
     <div class="col-sm-4 wow fadeInDown text-center">
       <img  class="rotate" src="img/icon/laptop.svg" alt="Generic placeholder image">
        <h3>Perl</h3>
        <p class="lead">
-		Yes! But not only Perl.
+		Yes! But not only Perl. Attendees and <a href="/barcelona2016/newtalk">speakers</a> from other communities are welcome. JavaScript, PHP, Ruby, Python, C, Java, Smalltalk, Linux, MacOS, Windows, Makers, Web Developers, Data Scientists, etc are invited. All sexes, sizes and ages. You only need to agree with our <a href="/barcelona2016/coc.html">CoC</a>.
```

Comment:
I know it's a complicated matter and maybe I'm getting the semantics wrong but I'm not sure if 'agree' is the right word here.

Personally, I'm not sure if I agree with the CoC (or with the fact that we have a CoC) and I don't think that it should be motive not to allow me in.

I'd probably say something like 'follow' or some other synonym.